tests/Unit/MakesAssertionsTest.php

```diff
@@ -1346,7 +1346,33 @@ public function test_assert_dont_see()
         }
     }
 
-    public function test_assert_see_in()
+//    public function test_assert_see_in()
```

Comment:
Duplicated, commented out, and renamed below when I thought more than one test would be needed. This should go away and keep the original test name, I guess.